resources/assets/build/webpack.config.js

```diff
@@ -1,217 +1,209 @@
+    performance: {
+        maxEntrypointSize: 512000,
+        maxAssetSize:        512000,
+    },
+    module: {
+        rules: [
+        {
+            enforce: 'pre',
+            test: /\.(js|s?[ca]ss)$/,
+            include: config.paths.assets,
+            loader: 'import-glob',
+        },
+        {
+            test: /\.js$/,
+            exclude: [/node_modules(?![/|\\](bootstrap|foundation-sites))/],
+            use: [
+            { loader: 'buble-loader', options: { objectAssign: 'Object.assign' } },
+            ],
+        },
+        {
+            test: /\.css$/,
+            include: config.paths.assets,
+            use: [
+            MiniCssExtractPlugin.loader,
+            { loader: 'css-loader', options: { sourceMap: config.enabled.sourceMaps } },
             {
-              loader: 'postcss', options: {
-                postcssOptions: { path: __dirname, ctx: config },
-                sourceMap: config.enabled.sourceMaps,
-              },
+                loader: 'postcss-loader', options: {
+                    postcssOptions: {
+                        config: path.join(__dirname, 'postcss.config.js'),
+                        ctx: config,
+                    },
+                    sourceMap: config.enabled.sourceMaps,
+                },
             },
-          ],
-        }),
-      },
-      {
-        test: /\.scss$/,
-        include: config.paths.assets,
-        use: ExtractTextPlugin.extract({
-          fallback: 'style',
-          use: [
-            { loader: 'cache' },
-            { loader: 'css', options: { sourceMap: config.enabled.sourceMaps } },
+            ],
+        },
+        {
+            test: /\.scss$/,
+            use: [
+            MiniCssExtractPlugin.loader,
+            { loader: 'css-loader', options: { sourceMap: config.enabled.sourceMaps } },
             {
-              loader: 'postcss', options: {
-                postcssOptions: { path: __dirname, ctx: config },
-                sourceMap: config.enabled.sourceMaps,
-              },
+                loader: 'postcss-loader', options: {
+                    postcssOptions: {
+                        config: path.join(__dirname, 'postcss.config.js'),
+                        ctx: config,
+                    },
+                    sourceMap: config.enabled.sourceMaps,
+                },
             },
-            { loader: 'resolve-url', options: { sourceMap: config.enabled.sourceMaps } },
+            { loader: 'resolve-url-loader', options: { sourceMap: config.enabled.sourceMaps } },
             {
-              loader: 'sass', options: {
-                sourceMap: config.enabled.sourceMaps,
-                sourceComments: true,
-              },
+                loader: 'sass-loader', options: {
+                    sassOptions: {
+                        sourceComments: true,
+                    },
+                    sourceMap: true, //config.enabled.sourceMaps, // false causes a resolve issue
```

Comment:
I wasn't able to reproduce the `sourceMap: false` when setting to `sourceMap: config.enabled.sourceMaps` and running `yarn dev` (which sets it to false aka `!inProduction`).

Can you test again and if it's working now for you make sure to change it back to pull from the config value?